services/provider/api/interfaces/interfaces.go

```diff
@@ -1,16 +1,16 @@
 package interfaces
 
 type StorageClientStatus interface {
-	// TODO: it was mistake not using full name of the field and we are just
-	// doing indirection for getters, change this interface after ensuring
-	// no client is using it
-	GetPlatformVersion() string
-	GetOperatorVersion() string
+	// all getters are implemented by proto itself and only setters need to
+	// be implemented
+	GetClientPlatformVersion() string
+	GetClientOperatorVersion() string
```

Comment:
Why the rename? we already are in the context of client reporting back?

Reply:
reinstated.